tools/cli/helpers/projectHelpers.js

```diff
@@ -26,6 +26,21 @@ export function allProjects() {
 	return projects;
 }
 
+/**
+ * Checks if a project name already is valid to use (if it doesn't already exist and isn't blank)
+ *
+ * @param {string} dir - the directory we're checking.
+ * @param {string} newName - the name of the new project we're checking.
+ *
+ * @returns {boolean} - return true if we're newName matches an existing project name.
+ */
+export function checkNameValid( dir, newName ) {
```

Comment:
Note for us to check out—what are the valid characters for a packagist project? Should we limit it to hyphen and alpha chars? Just strip invalid characters? Strip and prompt?

We can use this function to limit and pass to a new function to strip if we opt to do that.

Reply:
3397dd5